src/EndpointParser/parser_test.go

```diff
@@ -1,104 +1,104 @@
+// 	}
+// 	if endpoints[1].Summary != "PostUser" {
+// 		t.Errorf("The second endpoint should have summary PostUser but got %s", endpoints[1].Summary)
+// 	}
+// 	if endpoints[1].Description != "Post user data" {
+// 		t.Errorf("The second endpoint should have description Post user data but got %s", endpoints[1].Description)
+// 	}
+// 	if len(endpoints[1].Headers) != 1 {
+// 		t.Errorf("The second endpoint should have 1 header but got %d", len(endpoints[1].Headers))
+// 	}
+// 	if endpoints[1].Headers[0].Key != "Hello" {
+// 		t.Errorf("The second endpoint should have header Hello but got %s", endpoints[1].Headers[0].Key)
+// 	}
+// 	if endpoints[1].Headers[0].Description != "optional username" {
+// 		t.Errorf("The second endpoint should have header Hello with description optional username but got %s", endpoints[1].Headers[0].Description)
+// 	}
+// 	if endpoints[1].Headers[0].IsRequired != false {
+// 		t.Errorf("The second endpoint should have header Hello as optional but got %t", endpoints[1].Headers[0].IsRequired)
+// 	}
+// 	if endpoints[2].Method != "DELETE" {
+// 		t.Errorf("The third endpoint should have method DELETE but got %s", endpoints[2].Method)
+// 	}
+// 	if endpoints[2].Path != "/user" {
+// 		t.Errorf("The third endpoint should have path /user but got %s", endpoints[2].Path)
+// 	}
+// 	if endpoints[2].HandlerID != "controllers.DeleteUser" {
+// 		t.Errorf("The third endpoint should have handlerId controllers.DeleteUser but got %s", endpoints[2].HandlerID)
+// 	}
+// 	if endpoints[2].Summary != "DeleteUser" {
+// 		t.Errorf("The third endpoint should have summary DeleteUser but got %s", endpoints[2].Summary)
+// 	}
+// 	if endpoints[2].Description != "Delete user data" {
+// 		t.Errorf("The third endpoint should have description Delete user data but got %s", endpoints[2].Description)
+// 	}
+// 	if len(endpoints[2].Headers) != 2 {
+// 		t.Errorf("The third endpoint should have 2 header but got %d", len(endpoints[2].Headers))
+// 	}
+// 	if endpoints[2].Headers[0].Key != "Hello" {
+// 		t.Errorf("The third endpoint should have header Hello but got %s", endpoints[2].Headers[0].Key)
+// 	}
+// 	if endpoints[2].Headers[0].Description != "optional username" {
+// 		t.Errorf("The third endpoint should have header Hello with description optional username but got %s", endpoints[2].Headers[0].Description)
+// 	}
+// 	if endpoints[2].Headers[0].IsRequired != false {
+// 		t.Errorf("The third endpoint should have header Hello as optional but got %t", endpoints[2].Headers[0].IsRequired)
+// 	}
+// 	if endpoints[2].Headers[1].Key != "no" {
+// 		t.Errorf("The third endpoint should have header no but got %s", endpoints[2].Headers[1].Key)
+// 	}
+// 	if endpoints[2].Headers[1].Description != "test me please" {
+// 		t.Errorf("The third endpoint should have header no with description test me please but got %s", endpoints[2].Headers[1].Description)
+// 	}
+// }
```

Comment:
same question

Reply:
i said why in the description of this PR